osmaxx/excerptexport/models/export.py

```diff
@@ -164,7 +164,7 @@ def is_status_final(self):
 
     @property
     def can_be_deleted(self):
-        return self.is_status_final or self.update_overdue
+        return self.is_status_final or self.update_overdue()
```

Comment:
Good idea to include `is` in the name (whether this stays a method or becomes a property). I'm impartial between `update_is_overdue` and `is_update_overdue`.